.github/workflows/onRelease.yml

```diff
@@ -2,20 +2,31 @@ name: publish
 
 on:
   release:
-    types: [released]
-  # support manual release in case something goes wrong and needs to be repeated or tested
+    types: [published]
   workflow_dispatch:
     inputs:
       tag:
-        description: tag that needs to publish
+        description: prerelease-albatross
```

Comment:
This description does not need to be specific to prereleases. You could manually run this workflow with a tag (prerelease or not) to trigger a publish

Reply:
Got it, I updated it.